### PR TITLE
BZ-2107207- Documentation changes for SNR Operator config

### PR DESCRIPTION
**Pull request by abhatt-rh**
BZ-2107207: This PR updated issues reported in the BZ-2107207 related to the SNR config

Version(s): OpenShift 4.11+

Issue:
https://bugzilla.redhat.com/show_bug.cgi?id=2107207

Preview:

http://file.rdu.redhat.com/avbhatt/bz-2107207/nodes/nodes/eco-self-node-remediation-operator.html#understanding-self-node-remediation-operator-config_self-node-remediation-operator-remediate-nodes

### CODE REVIEW

**Comment by abhatt-rh** [comment]
Hi @prabinovRedhat Based on https://github.com/openshift/openshift-docs/pull/47363#discussion_r921147503 and https://github.com/openshift/openshift-docs/pull/47363#discussion_r921147831, you created 2107207. This PR addresses those comments. PTAL and approve the changes

**Review by pneedle-rh — COMMENTED** [review]
@abhatt-rh I have completed my review of this PR. Please let me know if you have any questions about my suggestions. Thanks!

File: modules/eco-self-node-remediation-operator-about.adoc [hunk 1]
```diff
@@ -13,7 +13,16 @@ The Self Node Remediation Operator minimizes downtime for stateful applications
 [id="understanding-self-node-remediation-operator-config_{context}"]
 == Understanding the Self Node Remediation Operator configuration
 
-The Self Node Remediation Operator creates the `SelfNodeRemediationConfig` CR with the name `self-node-remediation-config` in the Self Node Remediation Operator's namespace. You can edit this CR. However, you cannot create a new CR for the Self Node Remediation Operator.
+The Self Node Remediation Operator creates the `SelfNodeRemediationConfig` CR with the name `self-node-remediation-config` in the Self Node Remediation Operator's namespace. You can edit this CR. However, when you try to create a new CR for the Self Node Remediation Operator, the follow message in displayed in the logs:
```
[the remark this answers is not in the document]
Comment (abhatt-rh) [reply modules/eco-self-node-remediation-operator-about.adoc]: Done! Makes sense about the adding that information in a note.

**Comment by prabinovRedhat** [comment]
lgtm

**Comment by abhatt-rh** [comment]
@pneedle-rh Thank you for the review. I have addressed your comments. PTAL

**Pull request merged**